pkgs/development/ocaml-modules/ipaddr/default.nix

```diff
@@ -1,20 +1,24 @@
-{ stdenv, fetchurl, buildDunePackage, sexplib, ppx_sexp_conv }:
+{ stdenv, fetchurl, buildDunePackage, ocaml
+, macaddr, ounit, sexplib
+}:
 
 buildDunePackage rec {
   pname = "ipaddr";
-  version = "2.8.0";
+  version = "3.1.0";
 
   src = fetchurl {
```

Comment:
The `version` and `src` attributes should be inherited from `macaddr`.

Reply:
Ah yes, makes sense.